src/rules/or/out.jl

```diff
@@ -0,0 +1,8 @@
+@rule OR(:out, Marginalisation) (
+    m_in1::Bernoulli,
+    m_in2::Bernoulli
+) = begin
+    pin1, pin2 = mean(m_in1), mean(m_in2)
+
+    return Bernoulli(pin1 + pin2 - pin1 * pin2)
```

Comment:
Please double check this one, I end up with `pin1 + pin2 - 2 * pin1 * pin2`

Reply:
I ended up with ` pin1 + pin2 - pin1 * pin2 ` as well.  @bartvanerp Could it be that you omitted 'in1 = 1, in2=1'?

Reply:
Ah yeah you are absolutely right, accidentally did the computations for a XOR operation